html/js/elsinore.js

```diff
@@ -1473,6 +1475,7 @@ function showConfig() {
             var editBody = editModal.find(".modal-body");
             editBody.html(settingsHTML);
             editModal.find(".modal-footer").hide();
+            var editModalBody = editModal.find(".modal-body");
```

Comment:
on line 1475 there's already "editBody", the lines below this just need to be changed to use editBody. Could you do this so there's no additional jquery lookup?

Then I'll merge